README.md

```diff
@@ -1,2 +1,3 @@
 # tryit-jssip
-This project provides an webRTC client that can be used to connect to almost any public webRTC Server.
\ No newline at end of file
+This project provides an webRTC client that can be used to connect to almost any public webRTC Server.
+Note that is used a free Javascript library name jsSIP.
```

Comment:
There is a typo. Must be "named" instead of "name"

Reply:
Changed to "named". Thanks.